community/blog/2020-04-Releasing-Arcus-Templates-v0.2.0/README.md

```diff
@@ -0,0 +1,54 @@
+# Announcing Arcus Templates
+
+Recently we released v0.2.0 of the [Arcus.Templates](https://github.com/arcus-azure/arcus.templates/releases/tag/v0.2.0) project and is available on NuGet.
+This library will boost development for new projects with option-in/out of commonly used functionality.
+
+Take a look what theses templates have to offer!
+
+## Web API Template
+
+Version v0.1.0 only contained some basic functionality, exception handling, and an option to add a shared key authentication mechanism.
```

Comment:
This is a good list but more feels like release notes, what I would do is talk people through it.

For example: "We've improved the options to include authentication in our API by adding JWT & client certificate authentication.

Reply:
Yes, I've updated this.